### PR TITLE
use macOS build environment with Qt 5.12.3

### PR DESCRIPTION
**Pull request by Be-ing**

### CODE REVIEW

**Comment by Be-ing** [comment]
Woohoo it worked without any more weird hacks!

**Comment by Be-ing** [comment]
Now with KeyFinder :smile:

File: packaging/macos/build_environment [hunk 1]
```diff
@@ -1,2 +1,2 @@
-2.3-55d9a17-sdk10.15-macosminimum10.12-x86_64
-86c56d3aba24ec23923ad906f5a13084b0f784c955812448ecb43821fd907d50
+2.3-j00017-a52ffcc9-osx10.11-x86_64-release
```
Comment (uklotzde) [inline packaging/macos/build_environment:1]: Is `osx10.11` correct? I always struggle with these incomprehensible build environment names.

Comment (Be-ing) [reply packaging/macos/build_environment:1]: No, but the URL is what it is.

**Comment by Be-ing** [comment]
Ugh, doesn't work :cry: https://mixxx.discourse.group/t/macos-big-sur-issue-and-temporary-resolution/20257/142

**Comment by Be-ing** [comment]
I tried building on macOS 11.0, but the jobs are waiting in the queue before starting much longer than 10.15. I am guessing this is because most projects use `macos-latest` whereas there aren't many using the 10.15 VMs.

**Comment by Be-ing** [comment]
Apparently GH Actions no longer has the macOS 10.15 SDK available. We could use the macOS 11.0 SDK to make a new build environment, but we'd have to use a [fork of PortAudio](https://github.com/PortAudio/portaudio/pull/359) that supports this. There is no pressing need to bother doing that now that we have a working build environment from Jenkins that includes libKeyFinder and other updated dependencies. We can make a new build environment on GH Actions when there is a new PortAudio release.

**Comment by Be-ing** [comment]
CI passed. Merge?

**Pull request merged**